private_gpt/ui/ui.py

```diff
@@ -409,11 +410,30 @@ def _build_ui_blocks(self) -> gr.Blocks:
                         inputs=system_prompt_input,
                     )
 
+                def get_model_label() -> str | None:
+                    # Determine the model label based on PGPT_PROFILES env variable.
+                    pgpt_profiles = os.environ.get("PGPT_PROFILES")
+                    if pgpt_profiles == "ollama":
```

Comment:
You could cover more cases:
- For "local" you can show the settings().local.llm_hf_model_file
- For "openai" you can use the same as for "vllm"
- For "sagemaker" you can use "llm_endpoint_name"

Reply:
I've done another commit with these changes: https://github.com/imartinez/privateGPT/pull/1648/commits/133c1da13a6054541e1f0b307990db5e9b22f25d